src/components/Navbar.jsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import '../styles/Navbar.css';
+import 'boxicons';
 
 const Navbar = () => (
-  <nav className="nav-items">
-    <div className="logo">Bookstore CMS</div>
-    <ul className="nav-links">
-      <li><Link className="navlink" to="/">Books</Link></li>
-      <li><Link className="navlink" to="/Categories">Categories</Link></li>
+  <nav className="nav-container">
+    <ul className="nav-items">
+      <li className="nav-right">
+        <div className="logo">Bookstore CMS</div>
+        <ul className="nav-links">
+          <li><Link className="navlink" to="/">Books</Link></li>
+          <li><Link className="navlink org" to="/Categories">Categories</Link></li>
+        </ul>
+      </li>
+      <li className="user">
+        <box-icon type="solid" name="user" color="#0290ff" />
+      </li>
```

Comment:
- [ OPTIONAL] You did a great job styling the app as requested, I will recommend to center the icon inside the circle, your app will look perfect.